static/styles/rewards/pay.css

```diff
@@ -20,15 +20,29 @@
   font-weight: 400;
   text-transform: uppercase;
   letter-spacing: 2px;
+  box-sizing: border-box;
 }
 
+html {
+  display: block;
```

Comment:
Why

Reply:
I know it's implicitly defined from browser styling but I'm adding it in case someone works on it in the future. Because percentage values are inherited and the body tag percentage Val is relative/ driver from that if the HTML tag

Reply:
We need that to make the units inheritable from the viewport without having to use the "vw" measure unit